cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/MetricsAndLoggingCm.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource;
```

Comment:
This seems to be used exclusively by assembly operators. So why is it in resource operators package? Should it be moved to the assembly package?